### DISCUSSION TITLE
Set the max world size

### DISCUSSION THREAD

**Discussion by GoldenGamerLP** [Minestom/Minestom#1170] [Help] [answered]
How can I set the max world/instance size?

**Comment by dev-hydrogen** [comment, marked as answer]
Do you mean the world border? You can do Instance#getWorldBorder and then set the center and diameter of the world border
https://javadoc.minestom.net/net/minestom/server/instance/WorldBorder.html

**Reply by GoldenGamerLP** [reply]
No I meant the actual max world size, like by spigot, in the server.properties file

**Reply by dev-hydrogen** [reply]
> No I meant the actual max world size, like by spigot, in the server.properties file

The max-world-size option in server.properties only sets a maximum size that the world border can be changed to with commands. I'm not sure on what an "actual max world size" would be other than the world border size.

**Reply by GoldenGamerLP** [reply]
Oh okay, but how did he managed to get this:![Screenshot_20220618_183535.jpg](https://user-images.githubusercontent.com/65727384/174445727-3c1e5c3b-10e1-4a73-9d50-bc9f5fe7b1ee.jpg)

Via:![Screenshot_20220618_183708.jpg](https://user-images.githubusercontent.com/65727384/174445764-51ddad3c-a805-461d-a1ed-989d49c850bf.jpg)


Or did I get something wrong?

**Reply by dev-hydrogen** [reply]
Interesting. Apparently the max radius of the world border is indeed sent in a packet, but wiki.vg describes it as "Portal teleport boundary" and with a note of "Resulting coordinates from a portal teleport are limited to ±value. Usually 29999984." (https://wiki.vg/Protocol#Initialize_World_Border)

Minestom hard-codes this as 29999984 (https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/instance/WorldBorder.java#L42=)

**Reply by GoldenGamerLP** [reply]
So could you maybe make a fork plus a merge request? That would be great if it wasn't be hard-coded

**Reply by dev-hydrogen** [reply]
> So could you maybe make a fork plus a merge request? That would be great if it wasn't be hard-coded

I made an issue #1171 for it, but i'm not familiar enough with minestom to make a pull request to fix it 😛

**Reply by GoldenGamerLP** [reply]
Okay, but thanks for your afford!! 👍
